### PR TITLE
svirt: Add functionality of copying images to boot from

### PR DESCRIPTION
**Pull request by Soulofdestiny**

### CODE REVIEW

**Comment by coolo** [comment]
CC @mnowaksuse

File: consoles/sshVirtsh.pm [hunk 1]
```diff
@@ -322,6 +322,10 @@ sub add_disk {
                 $self->run_cmd("qemu-img create $file $size -f qcow2") && die "qemu-img create failed";
             }
         }
+        elsif ($args->{copy}) {
+            $file = "/var/lib/libvirt/images/$file";
+            $self->run_cmd("rsync -avL $args->{file} $file") && die "rsync failed";
```
Comment (richiejp) [inline consoles/sshVirtsh.pm]: Only a minor nit, but you could use single quotes for `rsync failed` because no substitution is intended.

Comment (coolo) [inline consoles/sshVirtsh.pm]: the -L is rather suprising when copying files.

Comment (Soulofdestiny) [inline consoles/sshVirtsh.pm]: yeah - this was a leftover from experimenting with symlinks

**Comment by okurz** [comment]
LGTM

**Comment by okurz** [comment]
oops, I merged the test, but can not merge this one … so coolo should work during the weekend to fix it

**Comment by coolo** [comment]
TGIF

**Pull request merged**